core/runtime/src/main/java/io/quarkiverse/langchain4j/runtime/devui/OpenWebUIJsonRPCService.java

```diff
@@ -0,0 +1,145 @@
+package io.quarkiverse.langchain4j.runtime.devui;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Map;
+
+import jakarta.enterprise.context.control.ActivateRequestContext;
+
+import org.eclipse.microprofile.config.ConfigProvider;
+import org.testcontainers.DockerClientFactory;
+
+import com.github.dockerjava.api.command.CreateContainerResponse;
+import com.github.dockerjava.api.command.InspectContainerResponse;
+import com.github.dockerjava.api.exception.NotFoundException;
+import com.github.dockerjava.api.model.Bind;
+import com.github.dockerjava.api.model.Container;
+import com.github.dockerjava.api.model.ContainerNetwork;
+import com.github.dockerjava.api.model.DeviceRequest;
+import com.github.dockerjava.api.model.HostConfig;
+import com.github.dockerjava.api.model.PortBinding;
+import com.github.dockerjava.api.model.Volume;
+
+@ActivateRequestContext
+public class OpenWebUIJsonRPCService {
+
+    public static final Runnable CLOSE_TASK = () -> {
+        OpenWebUIJsonRPCService service = new OpenWebUIJsonRPCService();
+        service.stopOpenWebUI();
+    };
+
+    private static final String CONTAINER_NAME_PREFIX = "quarkus-open-webui-";
+
+    public boolean isOpenWebUIRunning() {
+        var container = inspectOpenWebUIContainer();
+        return container != null && container.getState().getRunning();
+    }
+
+    public String getOpenWebUIUrl() {
+        var container = inspectOpenWebUIContainer();
+        if (container != null) {
+            ContainerNetwork network = container.getNetworkSettings().getNetworks().get("bridge");
+            if (network != null) {
```

Comment:
I don't understand podman too much, but here I get `network == null` so it can't find the port. The only network I get from `container.getNetworkSettings().getNetworks()` is named `podman`, not `bridge`

Reply:
I'll try to make it work regardless of docker/podman and fallback to the knwon port.